images/image.go

```diff
@@ -72,6 +73,13 @@ func SynchronousDelete() DeleteOpt {
 	}
 }
 
+func DeleteTarget(target *ocispec.Descriptor) DeleteOpt {
```

Comment:
Could we maybe add some godoc here (a la https://github.com/containerd/containerd/blob/74e205f1e7ae4ff8ccd17e4cf3068ec0c49fb49f/images/image.go#L66-L68)